src/python/grapl_analyzerlib/grapl_analyzerlib/test_utils/strategies/misc.py

```diff
@@ -9,9 +9,9 @@
 
 @st.composite
 def text_dgraph_compat(
-    draw: Callable[[st.SearchStrategy[str]], str], max_size: int = 128
+    draw: Callable[[st.SearchStrategy[str]], str], max_size: int = 64
```

Comment:
and went down to 64 because honestly it's not like the big length is gonna catch anything interesting